docs/DeveloperGuide.md

```diff
@@ -257,37 +257,36 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
-* has a need to manage a significant number of contacts
+* have large number of students to manage
+* accept fixed rate hour pay as a tutor, no matter which subject
 * prefer desktop apps over other types
 * can type fast
 * prefers typing to mouse interactions
 * is reasonably comfortable using CLI apps
 
-**Value proposition**: manage contacts faster than a typical mouse/GUI driven app
+**Value proposition**: 
+* manage students faster than a typical mouse/GUI driven app
+* keep track of the students' academic progress and specified requirements more efficiently
+
```

Comment:
Probably should follow the value proposition that we submitted

![image](https://user-images.githubusercontent.com/35413456/223061162-c5b89e94-5d2d-4dcf-8453-1305a833037c.png)